integration-tests/verify_oncotree_code_converter.py

```diff
@@ -0,0 +1,46 @@
+#!/bin/python
+
+import os
+import sys
+import linecache
+
+oncotree_code_converter_output = sys.argv[1]
+if not os.path.exists(oncotree_code_converter_output):
+    print "Error, specified file not found: " + oncotree_code_converter_output
+    sys.exit(1)
+
+REQUIRED_HEADERS = ["ONCOTREE_CODE", "CANCER_TYPE", "CANCER_TYPE_DETAILED"]
+
+# returns a dictionry from column name to list of column values
+# exits if one of the columns has empty value
+def get_required_columns(oncotree_code_converter_output):
+    with open(oncotree_code_converter_output) as file:
+        header_processed = False
+        header_index_mapping = {}
+        to_return = {header : [] for header in REQUIRED_HEADERS}
+        for line in file.readlines():
+            data = line.rstrip().split("\t")
+            if not header_processed:
+                header_index_mapping = {data[index] : index for index in range(len(data))}
+                header_processed = True
+                continue
+            for header in REQUIRED_HEADERS:
+                try:
+                    to_return[header].append(data[header_index_mapping[header]])
+                except KeyError:
```

Comment:
I don't think this will work, or rather it works just because your input file has no extra columns, you are not testing for "" values, which is what you would have if your header row looked like this "BLAH\tONCOTREE_CODE\tCANCER_TYPE\tCANCER_TYPE_DETAILED\tFOO" and your data row looked like this "junk\t\t\tmore junk".  The rstrip won't strip any tabs, and then you will split and get "" values for each of your required data fields.  I would probably just use the csv.DictReader and then test for None and "" blank values as well as the KeyErrors like you are.